todo_app/templates/index.html

```diff
@@ -10,8 +10,34 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
+      <form class="mb-4 mt-4" action="{{ url_for('addNewItem') }}" method="post">
+        <input type="text" name="addItemFormTitle" placeholder="Add new item">
```

Comment:
Minor but I'm always a bit wary of forms that don't have an explicit submit button - do we rely on all users knowing to just hit enter?